starter-code/src/functions-and-arrays.js

```diff
@@ -1,16 +1,52 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(a, b){
+  return Math.max(a, b);
+};
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+function findLongestWord(bla){
+    if(bla.length===0) return null;
+    let biggestWord = "";
+  bla.forEach(function(laengster){
+    if (laengster.length > biggestWord.length){
+      biggestWord = laengster;
+    }
+  })
+  return(biggestWord);
+}
+console.log(findLongestWord(words));
 
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-// Iteration #4: Calculate the average
+function sumArray(numArray){
+  let finalSum = 0;
+  for(let i = 0; i < numArray.length; i++){
+    finalSum += numArray[i];
+  }
+  return finalSum;
+}
+console.log(sumArray(numbers));
+
+
+// Iteration #4: Calculate the average 
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+function averageNumbers(numArray){
```

Comment:
how could you do it by reusing the sumArray() function?

Reply:
thx for hint @MarcoSantonastasi :) got it